HHS/Head-Start-TTADP
# Add focus to header on nav

Description of change
added focus to header on nav.

How to test
Verify that when moving through pages of a report, focus is set to the header.

Issue(s)

* closes #212 

Checklist

Meets issue criteria
Code tested
Meets accessibility standards (WCAG 2.1 Levels A, AA)
Documentation updated

    API methods
    Boundary and Data Flow Diagrams
    Architectural Decision Records written for major infrastructure decisions with the Nygard template
    OSCAL templates completed when security controls are implemented or modified
